### ISSUE TITLE
[BUG] DD_SERVICE_NAME not respected in Symfony integration

### ISSUE THREAD

**Issue by dcsg** [DataDog/dd-trace-php#846]
**Describe the bug**
After upgrading to 0.43.0 from 0.41.1 the DD_SERVICE_NAME stopped being respected and the service name is always `symfony`.

In our use case we have a `SetEnv` in the vhost to enforce the service name. I read the changelog and tried the the `DD_SERVICE_MAPPING` withouth success.

our ENV_VARS:

```
DD_AUTOFINISH_SPANS=true
DD_TRACE_ANALYTICS_ENABLED=true
DD_SYMFONY_ANALYTICS_ENABLED=true
DD_TRACE_HTTP_CLIENT_SPLIT_BY_DOMAIN=
DD_TRACE_URL_AS_RESOURCE_NAMES_ENABLED=false
DD_SERVICE_MAPPING=our_mappings
```

The  `DD_SERVICE_NAME` is set in the vhost.

Symfony version 4.4.7
Apache 2.4

**Comment by labbati** [comment]
@morrisonlevi any possibility we are now reading `DD_SERVICE_NAME` too early? (before the first request)? as being it in the vhost it will only be available at the first request.

**Comment by morrisonlevi** [comment]
I'm investigating now -- will report back later today.

**Comment by morrisonlevi** [comment]
I have confirmed the issue that was introduced in 0.43.0. PR #842 fixes this. I am going through the release process now and expect to release 0.43.1 in the next few hours.

**Comment by SammyK** [comment]
Hey @dcsg! Thanks so much for reporting this issue. The patch has been [released in 0.44.0](https://github.com/DataDog/dd-trace-php/releases/tag/0.44.0). Would you mind confirming that upgrading to 0.44.0 fixes the issue on your end?

**Comment by dcsg** [comment]
I can confirm this is solved in 0.44.0. Thanks
